dbld/rules

```diff
@@ -16,6 +16,7 @@ RELEASE_DIR=$(DBLD_DIR)/release
 VERSION=$(shell cat VERSION)
 TARBALL=$(BUILD_DIR)/syslog-ng-$(VERSION).tar.gz
 MODE=snapshot
+CONFIGURE_OPTS=--enable-manpages --with-python=3.5 --prefix=/install --with-docbook=/usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl
```

Comment:
Can we just use --with-python=3? Theoretically autotools and cmake should pick up the installed version automatically with it. Otherwise I'm afraid some specific versions are not available in some distributions.

Reply:
did just that. I have also added a CONFIGURE_ADD environment variable to customize environment variables during the bootstrap process.